package.json

```diff
@@ -89,17 +89,17 @@
     "@bbc/gel-foundations": "3.0.1",
     "@bbc/psammead-assets": "^1.0.2",
     "@bbc/psammead-brand": "^4.1.2",
-    "@bbc/psammead-caption": "^1.1.7",
+    "@bbc/psammead-caption": "^1.1.9",
     "@bbc/psammead-copyright": "^1.0.2",
     "@bbc/psammead-figure": "^1.0.2",
     "@bbc/psammead-headings": "^2.0.2",
     "@bbc/psammead-image": "^1.0.3",
     "@bbc/psammead-image-placeholder": "^1.0.1",
-    "@bbc/psammead-inline-link": "^1.1.0",
+    "@bbc/psammead-inline-link": "^1.1.2",
```

Comment:
This is now available at `1.1.3`. Is it worth updating to this?

Reply:
It needs to be bumped into the `paragraph` and `caption` as well otherwise we will pull in `inline-link` at 2 different versions due to the mismatch in the dependencies